test/server/server_test.cc

```diff
@@ -243,7 +243,8 @@ TEST_P(ServerInstanceImplTest, NoOptionsPassed) {
           Network::Address::InstanceConstSharedPtr(new Network::Address::Ipv4Instance("127.0.0.1")),
           hooks_, restart_, stats_store_, fakelock_, component_factory_,
           std::make_unique<NiceMock<Runtime::MockRandomGenerator>>(), thread_local_)),
-      EnvoyException, "unable to read file: ")
+      EnvoyException, "unable to read file: ");
```

Comment:
I'm not sure how server_test.cc got checked in without the semi-colon, because without it, the auto-formatter  scrambles  this test-method, or the 'format' test should fail.